tikv-configuration-file.md

```diff
@@ -964,6 +964,12 @@ Configuration items related to Raftstore.
 + Default value: `1MB`
 + Minimum value: `0`
 
+### `report-min-resolved-ts-interval`
+
++ Determines the minimum interval at which the resolved timestamp is reported to the PD leader. If this value is set to `0`, it means that the reporting is disabled.
++ Default value: `"1s"`
```

Comment:
The unit of this configuration is also seconds, so the smallest positive value is also 1s